src/main/java/net/mcreator/blockly/data/ToolboxType.java

```diff
@@ -23,5 +23,6 @@ public record ToolboxType(String name) {
 	public static ToolboxType AI_BUILDER = new ToolboxType("ai_builder");
 	public static ToolboxType PROCEDURE = new ToolboxType("procedure");
 	public static ToolboxType COMMAND = new ToolboxType("command");
+	public static ToolboxType FEATURE = new ToolboxType("feature");
 	public static ToolboxType EMPTY = new ToolboxType("empty");

```

Comment:
If you are not using it, maybe you should remove it as the Feature panel was the only one to still use it and Java plugins can now add toolboxes.

Reply:
It's still used by advancements

Reply:
Oh right, I forgot them